lib/beaker-pe/install/pe_utils.rb

```diff
@@ -1087,6 +1097,9 @@ def register_feature_flags!(opts)
         # @param [Hash] opts The Beaker options hash
         # @return [BeakerAnswers::Answers] the generated answers object
         def generate_installer_conf_file_for(host, hosts, opts)
+          possible_mco_enabled_setting = get_mco_setting(hosts)
+          opts[:answers] ||= {}
+          opts[:answers] = possible_mco_enabled_setting.merge(opts[:answers])
```

Comment:
Don't we still need a deep merge? This will overwrite `disable_mco` if `pe_install` exists in answers.

Reply:
I commented above, that should be 'pe_install::disable_mco', not a 'pe_install' hash...

Reply:
Yeah, that should address the merging issue.